skglm/tests/test_estimators.py

```diff
@@ -326,6 +326,37 @@ def test_Cox_sk_compatibility():
     check_estimator(CoxEstimator())
 
 
+@pytest.mark.parametrize("use_efron", [True, False])
+def test_cox_SLOPE(use_efron):
+    reg = 1e-2
```

Comment:
Nice, can.you just add a comment that this only checks SLOPE in the case where all alphas are equal because there is no alternative slope cox solver

Btw can you test if scipy.optimize.minimize gives the same result up to small errors, for a sequence alphas which is not constant ? That'd be a better test IMO

Reply:
yes sure, 
any hint at the gradient of SLOPE in this case?

Reply:
don't pass the gradient of slope, just let scipy estimate it with finite differences. it should work up to small numerical errors